src/argus_htmx/tailwindtheme/README.md

```diff
@@ -0,0 +1,6 @@
+# Files for tailwindcss
+
+`styles.css` and `tailwind.config.js` are automatically generated by the `tailwind_config` django
+managemet command. The source for these files are in `../templates/tailwind/`
```

Comment:
s/managemet/management/

The .. is relative to the README? More specificity would be nice.